diff --git a/train.py b/train.py
index bee22f7..a3e4c03 100644
--- a/train.py
+++ b/train.py
@@ -81,7 +81,7 @@ transform = torchvision.transforms.Compose([
 ])
 
 dataset = torchvision.datasets.ImageFolder(
-    root='/home/venom/repo/SKNets/COVID-19_Radiography_Dataset/Images', transform=transform)
+    root='/home/yoda/Documents/sknets/COVID-19_Radiography_Dataset/Images', transform=transform)
 train_ds, test_ds = random_split(dataset, [int(len(
     dataset)*0.8), int(len(dataset)*0.2)], generator=torch.Generator().manual_seed(42))
 train_loader = DataLoader(train_ds, batch_size=BATCH_SIZE, shuffle=True)
@@ -147,4 +147,4 @@ torch.save(model.state_dict(), model_path)
 
 if args.wandb:    
     wandb.save(model_path)
-    wandb.finish()
\ No newline at end of file
+    wandb.finish()
